test/common/grpc/google_async_client_cache_test.cc

```diff
@@ -0,0 +1,91 @@
+#include "common/grpc/google_async_client_cache.h"
+
+#include "test/mocks/server/factory_context.h"
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+namespace Envoy {
+namespace Grpc {
+namespace {
+
+class AsyncClientCacheTest : public testing::Test {
+public:
+  AsyncClientCacheTest() {
+    client_cache_singleton_ = std::make_unique<AsyncClientCacheSingleton>();
```

Comment:
you could put this in the member-var declaration initializer below and drop the ctor.

Reply:
done